CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ enable_testing()
 ##! Project options
 option(CLAWS_BUILD_TESTS "Build claws tests" ON)
 option(CLAWS_BUILD_EXAMPLES "Build claws examples" OFF)
-option(USE_PROJECT_IN_AN_IDE "Workaround for header-only libraries, put it to ON if you use CLION" ON)
+option(USE_PROJECT_IN_AN_IDE "Workaround for header-only libraries, put it to ON if you use CLION" OFF)
```

Comment:
I think `USE_PROJECT_IN_AN_IDE` sounds a little weird. Something like `IDE_BUILD` may be better.